experiments/20160805-mutagenesis/20160928-cpec-pol-assay

```diff
@@ -0,0 +1,27 @@
+## Goals
+
+- Identify polymerase plasmids with successfully mutated PB2 part 2 fragment from [CPEC library](./201609260transformation.md) for sequencing
+- Expecting luminescence results for mutants to be distinct from results for the template
+
+### Cell culture
+
+Followed [cell culture protocol](protocols/cell-culture.md) for preparing for polymerase assay.

```

Comment:
I think the relative path for the cell culture protocol is `../../protocols/cell-culture.md`.